### PR TITLE
Properly convert TKMOfflineAudioViewController

### PR DESCRIPTION
**Pull request by UInt2048**
This PR converts TKMOfflineAudioViewController to Swift for real. I managed to get it to work with the existing Light Untar dependency, and confirmed it works properly.

### CODE REVIEW

**Comment by UInt2048** [comment]
@davidsansome I apologise about the incident with yesterday's PR. I also took the testing I had performed in June and July for granted, but I had forgotten to test this one. It's now "basically functional" and would actually be functional if Light Untar wasn't erroring. Maybe when you review, something will jump out at you?

**Comment by UInt2048** [comment]
@davidsansome Ready for review. I have confirmed it works now!

File: ios/OfflineAudioViewController.swift [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  let fileManager: FileManager
+
+  required init?(coder: NSCoder) {
+    fileManager = FileManager()
+    super.init(coder: coder)
+  }
+
+  override func populateModel(_ model: TKMMutableTableModel) {
+    model.addSection("", footer: """
+    Download audio to your \(UIDevice.current.model) so it plays without\
```
Comment (davidsansome) [inline ios/OfflineAudioViewController.swift:87]: This is missing a space after "without" before the "/" - currently it renders as "withoutdelay" in the UI.

Comment (UInt2048) [reply ios/OfflineAudioViewController.swift]: Good catch! I fixed this now.

**Pull request merged**